core/state/statedb.go

```diff
@@ -44,6 +44,22 @@ type revision struct {
 	journalIndex int
 }
 
+type objectMutateType int
+
+const (
+	objectUpdate objectMutateType = iota
+	objectDelete
+)
+
+type objectPhase struct {
```

Comment:
How about 
- `objectPhase` -> `mutation`, and
- `objectMutateType` -> `mutationType`
- `objectUpdate` -> `update`
- `objectDelete` -> `deletion`
- `done` -> `applied`

?